examples/networking-hostroutes-playbook/README.md

```diff
@@ -0,0 +1,5 @@
+# networking-hostroutes-playbook
```

Comment:
Same comments as on the networking-backup-playbook README.

Reply:
As agreed above, I keep the playbook in the name, but updated the README to follow the cloud-aws example.